Makefile

```diff
@@ -49,13 +49,15 @@ utest:
 		github.com/Oneledger/protocol/data/governance \
 		github.com/Oneledger/protocol/data/rewards \
 		github.com/Oneledger/protocol/data/network_delegation \
+		github.com/Oneledger/protocol/data/evidence \
 		github.com/Oneledger/protocol/action/transfer \
 		github.com/Oneledger/protocol/serialize \
 		github.com/Oneledger/protocol/utils \
 		github.com/Oneledger/protocol/rpc \
 		github.com/Oneledger/protocol/identity \
 		github.com/Oneledger/protocol/app \
 		github.com/Oneledger/protocol/action/staking \
+		github.com/Oneledger/protocol/action/penalization \
```

Comment:
are we able to use a same package name, 'evidence'？

Reply:
Good point ,I think penalization makes more sense ,I can rename the data layer package

Reply:
> Good point ,I think penalization makes more sense ,I can rename the data layer package

Thanks. Take your time.

Reply:
actually renamed the action package to evidence ,since we already have the options names as evidenceoptions in kratos, it would be more uniform this way